addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
I'm not a big fan of big comment blocks either, but JSDoc on a function is my exception. I think it's far easier to enforce (in review at least) that all functions have JSDoc vs. trying to figure out when something is simple enough to not need it. For the simple ones, I like the `/** */` one liners for the same reason as @sglanzer. I'm not sure that multi-line paragraph comments are ever needed though. I guess the exception might be when discussing the top-level properties of a component. If it's talking about a block of code, I think the right move is to factor that block into a function and have that big comment block be the comment of the function. Comment blocks on chunks of code don't change with the chunk of code as much as comment blocks on functions change with the function. So you end up with less chance of stale comments (which are often worse than no comments) in that case.